Rakefile

```diff
@@ -14,7 +14,7 @@ end
 
 APP_RAKEFILE = File.expand_path("../spec/dummy/Rakefile", __FILE__)
 
-load 'rails/tasks/engine.rake' if File.exists?(APP_RAKEFILE)
+load 'rails/tasks/engine.rake' if File.exist?(APP_RAKEFILE)
```

Comment:
File.exists? is deprecated - it generates a warning when Ruby is run in verbose mode from Ruby 2.1.0 onwards.